source/options.rst

```diff
@@ -0,0 +1,90 @@
+.. _operator-configmaps:
+
+Changing PostgreSQL Options
+===========================
+
+You may require a configuration change for your application. PostgreSQL
+allows the option to configure the database with a configuration file.
+You can pass the `PostgreSQL configuration options <https://www.postgresql.org/docs/9.3/config-setting.html>`__
+in one of the following ways:
+
+* edit the ``deploy/cr.yaml`` file,
+* use a ConfigMap.
```

Comment:
If I use this, will my PostgreSQL configuration be rewritten or appended to existing ones? We need to mention this.